### ISSUE TITLE
Update Documentation Dependency

### ISSUE THREAD

**Issue by AdamKearn** [SteezCram/extDB3#19]
Hey,  the wiki needs an update here:  [https://github.com/SteezCram/extDB3/wiki/Instalation](https://github.com/SteezCram/extDB3/wiki/Instalation)

With the latest version of Debian 12 the `libtbb2` package is no longer is available.
I found using `apt-get install libtbbmalloc2` fixes this issue.

@SteezCram

**Comment by SteezCram** [comment]
Hello, sorry for the long response time. Yes, I will fix the documentation according to your feedback. Thank you so much!

**Comment by ksi9302** [comment]
FYI, On Ubuntu 20.04 (focal, and ARM64) I still had to install libtbb2 and libttmalloc2 is not available and extdb3 fails to load.
If there's any work around, I'd be very happy to know..

**Comment by SteezCram** [comment]
Hello, extDB3 is not available for ARM64 (unless you compile it yourself). Furthermore, I don't think Arma 3 provide an ARM64 version of the server.

**Comment by ksi9302** [comment]
Hey there,

Thanks for the reply. Well.. I kinda thought so.. (there goes my week T.T)
With arma server, I got it working following the below guide which suggests a workaround using box64 and box86.

https://www.reddit.com/r/valheim/comments/s1os21/create_your_own_free_dedicated_server/

**Comment by SteezCram** [comment]
By using box64 (or box86) you will emulate your x86/x86-64 program on your current platform: here ARM64. So you do not need ARM64 binaries but the x86 or x86-64 (depending on if you emulate 32 bits programs or 64 bits programs).

PS: you will suffer from performance penalties that can be severe due to the emulation process

**Comment by ksi9302** [comment]
That is right, surprisingly the performance is way better than I expected about average 20FPS with 100+ AIs and 4~6 players (I run modified TRGM) which I can double using HCs.

Saying that, I don't wanna go out without a fight, I'm not a C dev but if I wanted to compile it myself, where would I start?

**Comment by SteezCram** [comment]
You will not start any fight 😂, I am here to help you. The main problem is that it is impossible for an x86 program to run ARM64 code natively. If you use an emulator, it only allows your computer to run the program, but the main program is still x86 code. Arma 3 need to provide ARM64 binaries for the server, but I think it will be never the case since they use Steam to provide their server and Steam does not support ARM64 itself.

**Comment by ksi9302** [comment]
Ahh.. then I cannot put up any fight can I.. 😂 Thanks for the help anyways
🙏
